browser/js/orders/templates/order-detail-info.html

```diff
@@ -1,47 +1,35 @@
-<div class='panel panel-default'>
```

Comment:
Oh shit that was removed by mistake when i was fixing the merge conflicts on my branch, I'll fix it now.